pahfit/base.py

```diff
@@ -377,6 +380,9 @@ def save(self, obs_fit, filename, outform):
             Currently using the input data file name.
         outform : string
             Sets the output file format (ascii, fits, csv, etc.).
+        strength_calc : bool
+            Imports the feature_strength module
```

Comment:
This import is done automatically.  I reword this description to say what this code is doing.  Like: "compute the integrated dust and gas line strengths."  That is what this code is doing, right?